### PR TITLE
[PaddlePaddle] add paddle demo: Mask classification

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add demo for mask classification

### CODE REVIEW

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+   "outputs": [],
+   "source": [
+    "NDList processImageInput(NDManager manager, Image input, float shrink) {\n",
+    "            NDArray array = input.toNDArray(manager);\n",
+    "            Shape shape = array.getShape();\n",
+    "            array = NDImageUtils.resize(\n",
+    "                            array, (int) (shape.get(1) * shrink), (int) (shape.get(0) * shrink));\n",
+    "            array = array.transpose(2, 0, 1).flip(0); // HWC -> CHW BGR -> RGB\n",
+    "            NDArray mean = manager.create(new float[] {104f, 117f, 123f}, new Shape(3, 1, 1));\n",
+    "            array = array.sub(mean).mul(0.007843f); // normalization\n",
```
Comment (stu1130) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:119]: maybe use NDimageUtils.normalize here

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Criteria<Image, DetectedObjects> criteria =\n",
+    "        Criteria.builder()\n",
+    "                .optApplication(Application.CV.OBJECT_DETECTION)\n",
+    "                .setTypes(Image.class, DetectedObjects.class)\n",
+    "                .optArtifactId(\"face_detection\")\n",
+    "                .optTranslator(new FaceTranslator(0.5f, 0.7f))\n",
+    "                .optFilter(\"flavor\", \"server\")\n",
```
Comment (stu1130) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:233]: what does "server" here mean?

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+    "%mavenRepo snapshots https://oss.sonatype.org/content/repositories/snapshots/\n",
+    "\n",
+    "%maven ai.djl:api:0.9.0-SNAPSHOT\n",
+    "%maven ai.djl.paddlepaddle:paddlepaddle-model-zoo:0.9.0-SNAPSHOT\n",
+    "%maven ai.djl.paddlepaddle:paddlepaddle-native-auto:2.0.0-SNAPSHOT\n",
+    "%maven org.slf4j:slf4j-api:1.7.26\n",
+    "%maven org.slf4j:slf4j-simple:1.7.26\n",
+    "%maven net.java.dev.jna:jna:5.3.0\n",
+    "// second engine to do preprocessing and postprocessing\n",
+    "%maven ai.djl.pytorch:pytorch-engine:0.9.0-SNAPSHOT\n",
```
Comment (roywei) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:36]: some background intro for why use anther engine to pre/post processing?

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 4]
```diff
@@ -0,0 +1,382 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Face Mask Detection using PaddlePaddle\n",
+    "\n",
+    "In this tutorial, we will be using pretrained PaddlePaddle model from PaddleHub to do mask detection on the sample image. To complete this procedure, there are two steps needs to be done:\n",
```
Comment (roywei) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:9]: link to paddle hub?

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+    "                    prob.add((double) probabilities[i]);\n",
+    "                    boxes.add(\n",
+    "                            new Rectangle(\n",
+    "                                    array[2], array[3], array[4] - array[2], array[5] - array[3]));\n",
+    "                }\n",
+    "            }\n",
+    "            return new DetectedObjects(names, prob, boxes);\n",
+    "}\n",
+    "\n",
+    "NDArray fakeOutput = NDManager.newBaseManager().create(new float[]{1f, 0.99f, 0.1f, 0.1f, 0.2f, 0.2f}, new Shape(1, 6));\n",
```
Comment (roywei) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:165]:
```suggestion
    "NDArray tempOutput = NDManager.newBaseManager().create(new float[]{1f, 0.99f, 0.1f, 0.1f, 0.2f, 0.2f}, new Shape(1, 6));\n",
```

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb [hunk 6]
```diff
@@ -0,0 +1,382 @@
...
+    "DetectedObjects testBox = processImageOutput(new NDList(fakeOutput), Arrays.asList(\"Not Face\", \"Face\"), 0.7f);\n",
```
Comment (roywei) [inline jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:166]:
```suggestion
    "DetectedObjects testBox = processImageOutput(new NDList(tempOutput), Arrays.asList(\"Not Face\", \"Face\"), 0.7f);\n",
```

**Review by roywei — COMMENTED** [review]
LGTM. minor fixes

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb (hunk 2 above)
Comment (lanking520) [reply jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:235]: The server based model

File: jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb (hunk 1 above)
Comment (lanking520) [reply jupyter/paddlepaddle/face_mask_detection_paddlepaddle.ipynb:119]: normailize applies std, where this one use multiplication instead

**Pull request merged**